docs/settingUp.md

```diff
@@ -32,12 +39,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification:** Check your Gradle folder with the command `./gradlew printUserHomeDir`. A folder named `appengine-sdk` should be present.
```

Comment:
The phrasing is not accurate. The command `./gradlew printUserHomeDir` is _not_ for checking the Gradle folder; it simply prints the location. The previous description, "Check your Gradle folder (the directory can be found with the command `./gradlew printUserHomeDir`).", is more accurate for all purpose.